kbcstorage/retry_requests.py

```diff
@@ -0,0 +1,29 @@
+import time
+import requests
+
+MAX_RETRIES = 11
```

Comment:
Why 10 retries? That means 17 minutes request if the service is down completely.

Reply:
http://backoffcalculator.com/?interval=1&attempts=10&rate=2

Reply:
I don't think we have 10 retries in SAPI, do we?

Reply:
https://github.com/keboola/storage-api-php-client/blob/master/src/Keboola/StorageApi/Client.php#L82

Reply:
😱

Reply:
hotovson